BlockEQ/View Controllers/Trade/MyOffersViewController.swift

```diff
@@ -19,30 +21,103 @@ class MyOffersViewController: UIViewController {
     }
     
     func setupView() {
-        let offersCellNib = UINib(nibName: OffersCell.cellIdentifier, bundle: nil)
-        tableView.register(offersCellNib, forCellReuseIdentifier: OffersCell.cellIdentifier)
+        let offerNib = UINib(nibName: OffersCell.cellIdentifier, bundle: nil)
+        tableView.register(offerNib, forCellReuseIdentifier: OffersCell.cellIdentifier)
```

Comment:
If you conform your `OffersCell` to `ReusableView, NibLoadableView`, this can be simplified to `tableView.registerCell(type: OffersCell.self)`